plugins/source/datadog/main.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+import (
+	"github.com/cloudquery/cloudquery/plugins/source/datadog/resources/plugin"
+	"github.com/cloudquery/plugin-sdk/serve"
+)
+
+const sentryDSN = "https://99f66f1a627f48deb66e49a25d6028a6@o1396617.ingest.sentry.io/6747628"
```

Comment:
```suggestion
const sentryDSN = "https://a0c2570b96264748a6759bb62e8cdef5@o1396617.ingest.sentry.io/4504220208267264"
```

Here is a new DSN to use for Datadog